tests/test_wiper.py

```diff
@@ -0,0 +1,53 @@
+import getpass
+import os
+import shutil
+import tempfile
+import unittest
+from dbb_buffer_mngr import Wiper
+
+
+class WiperTestCase(unittest.TestCase):
+    """Test remote cleaner.
+    """
+
+    def setUp(self):
+        self.dir = tempfile.mkdtemp()
+        self.user = getpass.getuser()
+        self.host = "localhost"
+
+    def tearDown(self):
+        shutil.rmtree(self.dir)
+
+    def testInvalidConfig(self):
+        config = dict()
+        args = [config]
+        self.assertRaises(ValueError, Wiper, *args)
+
+    def testEmpty(self):
+        subdir = tempfile.mkdtemp(dir=self.dir)
+
+        config = dict(user=self.user, host=self.host, staging=self.dir)
+        cmd = Wiper(config)
+        cmd.run()
+
+        dirs = []
+        for topdir, subdirs, files in os.walk(self.dir):
+            for d in subdirs:
+                dirs.append(d)
+        self.assertEqual(len(dirs), 0)
+
+    def testNonEmpty(self):
+        subdir = tempfile.mkdtemp(dir=self.dir)
+        files = dict([tempfile.mkstemp(dir=os.path.join(self.dir, subdir))])
+        for fd in files:
+            os.close(fd)
+
+        config = dict(user=self.user, host=self.host, staging=self.dir)
+        cmd = Wiper(config)
+        cmd.run()
+
+        dirs = []
+        for top, subs, files in os.walk(self.dir):
+            for d in subs:
```

Comment:
The intention behind this test is to check if the command does not remove directories containing files. I don't really care about the file I created. I'm interested if the directory is still there after the command run.

Reply:
I got myself confused about what Wiper does.  I thought this test was checking to make sure it was removing everything given a non-empty directory when Wiper is only supposed to get rid of empty directories.  Checking the assertEqual function at the end I see that the test does only succeed if it Wiper didn't delete the non-empty directory.   It would be helpful to have a short comment describing what each test is doing at the top of each test.